packages/beacon-node/src/chain/initState.ts

```diff
@@ -90,7 +90,7 @@ export async function initStateFromEth1({
     logger,
     signal,
     pendingStatus:
-      statePreGenesis && depositTree !== undefined && lastProcessedBlockNumber != null
+      statePreGenesis && depositTree !== undefined && lastProcessedBlockNumber !== null
```

Comment:
`depositTree` is ` ListCompositeTreeViewDU<ByteVectorType>`. Compiler ensures it can't be null or undefined so this check is actually not necessary.

type of `lastProcessedBlockNumber` is `number | null`. Won\t be `undefined` so strict checking against `null` is safe